scripts/py_matter_idl/matter/idl/tests/outputs/large_lighting_app/cpp-app/ember-cluster-config/AccessControl.h

```diff
@@ -0,0 +1,33 @@
+// DO NOT EDIT - Generated file
+//
+// Application configuration for AccessControl based on EMBER configuration
+// from inputs/large_lighting_app.matter
+#pragma once
+
+#include <app-common/zap-generated/cluster-enums.h>
+#include <app/util/cluster-config.h>
+
+#include <array>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace AccessControl {
+namespace EmberApplicationConfig {
+
+using FeatureBitmapType = uint32_t;
```

Comment:
Access Control has a feature map defined, no?  Why is this ending up as uint32_t?